test/integration/sftp/controls/sftp.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+control 'SFTP configuration' do
+  title 'should match desired lines'
+
+  describe file('/etc/proftpd/sftp.conf') do
+    it { should be_file }
+    its('content') { should include '## File managed by Salt' }
+    its('content') { should include 'SFTPEngine' }
+    its('content') { should include 'SFTPLog' }
+    its('content') { should include 'SFTPOptions' }
+    its('content') { should include 'SFTPHostKeyrsa' }
+    its('content') { should include 'SFTPAuthMethods' }
+    its('content') { should include 'SFTPAuthorizedUserKeys' }
+    its('content') { should include 'Port 	4000' }
```

Comment:
the number of spaces between `Port` and `4000` is weird, any reason to that?

Reply:
this is what we expect to find in the final sftp.conf file.
I tried to reduce these spaces and the check fails